CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 3.17)
 set (CMAKE_CXX_STANDARD 17)
 set (CMAKE_CXX_STANDARD_REQUIRED ON)
 
-project(olympia CXX)
+project(olympia C CXX)
```

Comment:
I think we don't need the C anymore -- fixed HDF5 find package.

Reply:
Great! I had to add it for hdf5 indeed